recipes/default.rb

```diff
@@ -17,22 +17,35 @@
 # limitations under the License.
 #
 
-package 'cron' do
-  package_name case node['platform_family']
-               when 'rhel', 'fedora'
-                 node['platform_version'].to_f >= 6.0 ? 'cronie' : 'vixie-cron'
-               when 'solaris2'
-                 'core-os'
-               when 'arch'
-                 'cronie'
-               when 'gentoo'
-                 'vixie-cron'
-               end
+node['cron']['package_name'].each do |pkg|
+  package pkg
 end
 
-service 'cron' do
```

Comment:
This looks like a mistake here.  I fixed it in master, but it caused compile failures since the resource isn't actually defined.